WcaOnRails/db/structure.sql

```diff
@@ -1507,6 +1508,7 @@ CREATE TABLE `wcif_extensions` (
 /*!50001 SET character_set_results     = utf8mb4 */;
 /*!50001 SET collation_connection      = utf8mb4_unicode_ci */;
 /*!50001 CREATE ALGORITHM=UNDEFINED */
+/*!50013 DEFINER=`root`@`%` SQL SECURITY DEFINER */
```

Comment:
This was autogenerated. Unsure if it should have been committed.

Reply:
Based on the [mass test failures](https://github.com/thewca/worldcubeassociation.org/actions/runs/3972046377/jobs/6809564177), should not have been committed.

Removed. (Changed squashed into first commit.)